src/main/java/com/epam/ta/reportportal/core/item/UpdateTestItemHandler.java

```diff
@@ -84,4 +85,6 @@ OperationCompletionRS updateTestItem(ReportPortalUser.ProjectDetails projectDeta
 	 * @param projectId Project id
 	 */
 	void resetItemsIssue(List<Long> itemIds, Long projectId);
+
+	OperationCompletionRS bulkUpdateAttributes(BulkUpdateItemAttributeRQ bulkUpdateRq, ReportPortalUser.ProjectDetails projectDetails);
```

Comment:
it consists not only of attributes, so the naming should be according to the logic.
for example bulkItemInfoUpdate
the same for BulkUpdateObj rq.
Add a few words of documentation to the method